tests/unit/index-nodetest.js

```diff
@@ -70,7 +70,68 @@ describe('base plugin', function() {
       plugin.log('foo', {verbose: true});
       assert.deepEqual(verboseUi.messages, ['\u001b[34m|    \u001b[39m']);
     });
-
   });
 
+  describe('plugin helper', function() {
+    it('provides access to the oringal default values', function() {
+      var Plugin = Subject.extend({
+        requiredConfig: ['bar'],
+        defaultConfig: {
+          foo: function(context) {
+            return context.foo;
+          }
+        }
+      });
+
+      var plugin = new Plugin({
+        name: 'blah'
+      });
+
+      var context = {
+        foo: 'foo',
+        config: {
+          blah: {
+            foo: function(context, pluginHelper) {
+              return pluginHelper.readConfigDefault('foo') + 'foo';
+            },
+            bar: function(context, pluginHelper) {
+              return pluginHelper.readConfigDefault('bar') + 'bar';
+            }
+          }
+        }
+      };
+
+      plugin.beforeHook(context);
+      assert.equal(plugin.readConfig('foo'), 'foofoo');
+      assert.equal(plugin.readConfig('bar'), 'undefinedbar');
+    });
+
+    it('allows the implementer to add things to the pluginHelper', function() {
+      var Plugin = Subject.extend({
+        defaultConfig: { foo: 'foo' }
+      });
+
+      var plugin = new Plugin({
+        name: 'blah',
+
+        pluginHelper: function(context) {
+          return { bar: context.foo };
+        }
+      });
+
+      var context = {
+        foo: 'bar',
+        config: {
+          blah: {
+            foo: function(context, pluginHelper) {
+              return pluginHelper.readConfigDefault('foo') + pluginHelper.bar;
```

Comment:
As all relevant variables/properties of this test are called `foo` and `bar` I found it pretty hard to follow what's going on in this test. This may be just me but imo it would be easier to follow if the property on the context that can be accessed via the property configured on pluginHelper by the implementor would be called differently than the default configuration option we access here via `pluginHelper.readConfigDefault`